var/spack/repos/builtin/packages/conduit/package.py

```diff
@@ -37,7 +37,11 @@ class Conduit(Package):
     url      = "https://github.com/LLNL/conduit/releases/download/v0.3.0/conduit-v0.3.0-src-with-blt.tar.gz"
     git      = "https://github.com/LLNL/conduit.git"
 
-    version('master', branch='master', submodules=True, preferred=True)
+    version('develop', branch='develop', submodules=True, preferred=True)
+    # note: the main branch in conduit was renamed to develop, this next entry
+    # is to bridge any spack dependencies that are still using the name master
+    version('master', branch='develop', submodules=True)
```

Comment:
Why not fix spack dependencies that are still using the name master? I would rather be consistent.

Reply:
I don't think I have the visibility to tackle that with certainty.  It could be specified in folks `package.yaml` files, etc.

Reply:
@adamjstewart  -- can we get this merged in? Folks that were expecting to stay up-to-date with the main branch, will not longer be able to use spack to do so